drivers/serial/Kconfig

```diff
@@ -173,6 +173,20 @@ config SERIAL_TERMIOS
 		If this is not defined, then the terminal settings (baud, parity, etc).
 		are not configurable at runtime; serial streams cannot be flushed, etc..
 
+config TTY_FORCE_PANIC
```

Comment:
I think it is more appropriate to add these options below `CONFIG_DEBUG_FEATURES`, in a new option "Serial Debug Features".

Reply:
Yes, I agree! If it is used in production (final product) it should be used to get more information to hack the system. I used similar approach in the past to gather more information from a Sony TV running Linux: https://acassis.wordpress.com/2014/10/08/more-sony-kdl-32ex405-logs/
We want NuttX to be harder to hack!

Reply:
Good suggestion, and many thanks, TTY_FORCE_PANIC depends on DEBUG_FEATURES.